apps/twap/src/app/utils/getApiBaseUrl.ts

```diff
@@ -0,0 +1,15 @@
+import { SupportedChainId } from '@cowprotocol/cow-sdk';
+
+const COW_API_BASE_URL = 'https://api.cow.fi';
+
+const CHAIN_ID_TO_NAME: Record<SupportedChainId, string> = {
+  1: 'mainnet',
```

Comment:
Same comment as [here](https://github.com/cowprotocol/cow-web-services/pull/8/files#r1265224178)

Reply:
Great point, updated it